modules/combined/examples/phase_field-mechanics/LandauPhaseTrans.i

```diff
@@ -0,0 +1,301 @@
+    variable = eta1
+    args = 'eta2'
+    f_name = F
+  [../]
+  [./eta_bulk2]
+    type = AllenCahn
+    variable = eta2
+    args = 'eta1'
+    f_name = F
+  [../]
+  [./eta_interface1]
+    type = ACInterface
+    variable = eta1
+    kappa_name = kappa_eta
+  [../]
+  [./eta_interface2]
+    type = ACInterface
+    variable = eta2
+    kappa_name = kappa_eta
+  [../]
+
+  [./deta1dt]
+    type = TimeDerivative
+    variable = eta1
+  [../]
+  [./deta2dt]
+    type = TimeDerivative
+    variable = eta2
+  [../]
+[]
+
+#
+# Try visualizing the stress tensor components as done in Conserved.i
+#
+
+[Materials]
+  [./consts]
+    type = GenericConstantMaterial
+    prop_names  = 'L kappa_eta'
+    prop_values = '1 1'
+  [../]
+
+  [./chemical_free_energy]
+    type = DerivativeParsedMaterial
+    f_name = Fc
+    args = 'eta1 eta2'
+    constant_names = 'A2 A3 A4'
+    constant_expressions = '0.2 -12.6 12.4'
+    function = A2/2*(eta1^2+eta2^2)+A3/3*(eta1^3+eta2^3)+A4/4*(eta1^2+eta2^2)^2
+    enable_jit = true
+    derivative_order = 2
+  [../]
+
+  [./elasticity_tensor]
+    type = ComputeElasticityTensor
+    C_ijkl = '700 300 300 700 300 700 300 300 300'
+    fill_method = symmetric9
+  [../]
+
+  [./stress]
+    type = ComputeLinearElasticStress
+  [../]
+
+  [./var_dependence1]
+    type = DerivativeParsedMaterial
+    f_name = var_dep1
+    args = 'eta1'
+    function = eta1
+    enable_jit = true
+    derivative_order = 2
+  [../]
+  [./var_dependence2]
+    type = DerivativeParsedMaterial
+    f_name = var_dep2
+    args = 'eta2'
+    function = eta2
+    enable_jit = true
+    derivative_order = 2
+  [../]
+
+  [./eigenstrain1]
+    type = ComputeVariableEigenstrain
+    eigen_base = '0.1 -0.1 0 0 0 0'
+    prefactor = var_dep1
+    args = 'eta1'
+    eigenstrain_name = eigenstrain1
+  [../]
+
+  [./eigenstrain2]
+    type = ComputeVariableEigenstrain
+    eigen_base = '-0.1 0.1 0 0 0 0'
+    prefactor = var_dep2
+    args = 'eta2'
+    eigenstrain_name = eigenstrain2
+  [../]
+
+  [./strain]
+    type = ComputeSmallStrain
+    displacements = 'disp_x disp_y'
+    eigenstrain_names = 'eigenstrain1 eigenstrain2'
+  [../]
+
+  [./elastic_free_energy]
+    type = ElasticEnergyMaterial
+    f_name = Fe
+    args = 'eta1 eta2'
+    derivative_order = 2
+  [../]
+
+  [./totol_free_energy]
+    type = DerivativeSumMaterial
+    f_name = F
+    sum_materials = 'Fc Fe'
+    args = 'eta1 eta2'
+    derivative_order = 2
+  [../]
+[]
+
+[AuxVariables]
+  [./sigma11_aux]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./sigma12_aux]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
+[AuxKernels]
+  [./matl_sigma11]
+    type = RankTwoAux
+    rank_two_tensor = stress
+    index_i = 0
+    index_j = 0
+    variable = sigma11_aux
+  [../]
+  [./matl_sigma22]
+    type = RankTwoAux
+    rank_two_tensor = stress
+    index_i = 0
+    index_j = 1
+    variable = sigma12_aux
+    execute_on = 'timestep_end initial'
+  [../]
+[]
```

Comment:
This is probably all copy pasted from the existing example (which I'm actually updating right now). There is a more elegant/compact way of doing this now.